scripts/process_schemas.py

```diff
@@ -344,10 +366,15 @@ def main():
         help="The file to output the protocol definitions to.")
     parser.add_argument(
         "version",
-        help="The tagged git release to process, e.g., v0.5.1")
+        help="The tagged git release to process, e.g., v0.5.1. "
+        "Ignored if --inputSchemasDirectory is specified.")
     parser.add_argument(
         "--avro-tools-jar", "-j",
         help="The path to a local avro-tools.jar", default=None)
+    parser.add_argument(
+        "--inputSchemasDirectory", "-i",
+        help="Path to local schemas directory. If none, uses GitHub.",
```

Comment:
@jeromekelleher That's a good question. Currently, it's requiring the equivalent of the top-level directory of the cloned `ga4gh/schemas` project - so it assumes that there's a `src/main/resources/avro` subdirectory structure inside it. But on second thought, that's kind of a weird setup, which I guess is why you asked in the first place: Why shouldn't I just have this be the actual directory that contains the `.avdl` files (so in the case of a local clone of the `ga4gh/schemas` project located at `$LOCAL_GA4GH_SCHEMAS`, the argument to pass would then be `-i $LOCAL_GA4GH_SCHEMAS/src/main/resources/avro`.